packages/addon/src/components/DeveloperOptionsSelector.tsx

```diff
@@ -0,0 +1,149 @@
+// this file is based upon similar code by https://github.com/hipstersmoothie
+
+import React, { useState } from "react";
+import { API } from "@storybook/api";
+import {
+    IconButton,
+    WithTooltip,
+    TooltipLinkList,
+    Icons
+} from "@storybook/components";
+
+import { ACTION_EVENT_NAME } from "@storybook/native-controllers";
+import { EmulatorActions } from "@storybook/native-types";
+
+export interface DeveloperOptionsSelectorProps {
+    api: API;
+}
+
+export default ({ api }: DeveloperOptionsSelectorProps) => {
+    const [options, setOptions] = useState([
+        {
+            id: "layout_bounds",
+            name: "Show layout bounds",
+            value: false
+        },
+        {
+            id: "profile_gpu_rendering",
+            name: "Profile GPU rendering",
+            value: false
+        },
+        {
+            id: "over_draw",
+            name: "Show Overdraw",
+            value: false
+        }
+        // Not working as expected
+        // {
+        //     id: "dont_keep_activities",
+        //     name: "Don't keep activities",
+        //     value: false,
+        // },
+    ]);
+
+    const toggleShowLayoutBounds = (enabled: boolean) => {
+        api.getChannel()?.emit(
+            ACTION_EVENT_NAME,
+            EmulatorActions.showLayoutBounds,
+            null, // location
```

Comment:
Can you remove the `// location` comments in this file?

Reply:
emit accepts ordered  arguments , the comments indicates the argument type and position